src/pipelines/similarity_baseline_json.py

```diff
@@ -0,0 +1,131 @@
+import sys
+import spacy
+from spacy.attrs import *
+from sklearn.feature_extraction import DictVectorizer
+from scipy import spatial
+import json
+from collections import namedtuple
+
+def parseJSON(fileName):
+   
+    '''
+    Purpose - Parses a JSON file containing document data. 
+    Input - File name with JSON data
+    Output - A set of cluster IDs, a dict of document arrival order and an array of the parsed JSON document data
+        
+    The JSON data schema for ingesting documents into Pythia is:
+    corpus = name of corpus 
+    cluster_id = unique identifier for associated cluster
+    post_id = unique identifier for element (document, post, etc)
+    order = int signifying order item was received 
+    body_text = text of element (document, post, etc)
+    novelty = boolean assessment of novelty 
+    
+    Function returns an array of dictionaries containing data in JSON schema 
+    '''
+           
+    documentData = []
+    allClusters = set()
+    lookupOrder = {}
+    i = 0
+    
+    # Read JSON file line by line and retain stats about number of clusters and order of objects 
+    with open(fileName,'r') as dataFile:
+        for line in dataFile: 
+            parsedData = json.loads(line)
+            allClusters.add(parsedData["cluster_id"])
+            lookupOrder[str(parsedData["cluster_id"]) + "_" + str(parsedData["order"])] = i
```

Comment:
Do we know that this wouldn't break for clusters large than 9? ( "1_1", "1_10", "1_9", etc.)? just more evidence that we should move asap to a proper database-type system.